src/libs/ReportUtils.ts

```diff
@@ -1075,6 +1075,21 @@ function findLastAccessedReport(
     return adminReport ?? sortedReports.at(-1) ?? null;
 }
 
+/**
+ * Whether the provided report has transactions
+ */
+function hasTransactions(reportID: string | undefined): boolean {
+    const transactions = TransactionUtils.getAllReportTransactions(reportID);
```

Comment:
We shouldn't get all the transactions and check, we should only check if the report has any transaction? @rayane-djouah

Reply:
Something like this should be fine. Kindly have good naming and only money transactions considered.